Dockerfile

```diff
@@ -2,30 +2,41 @@ FROM debian:jessie
 
 MAINTAINER nine.ch Engineering <engineering@nine.ch>
 
-ENV FILEBEAT_VERSION=5.2.1 \
-    FILEBEAT_SHA1=694fe12e56ebf8e4c4b11b590cfb46c662e7a3c1
+ARG FILEBEAT_VERSION
+ENV FILEBEAT_VERSION=${FILEBEAT_VERSION:-"5.3.0"}
+ARG FILEBEAT_SHA1
+ENV FILEBEAT_SHA1=${FILEBEAT_SHA1:-"c6f56d1a938889ec9f5db7caea266597f625fcc1"}
 
+# update and install wget
 RUN set -x && \
-  apt-get update && \
-  apt-get install -y wget && \
-  wget https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-${FILEBEAT_VERSION}-linux-x86_64.tar.gz -O /opt/filebeat.tar.gz && \
-  cd /opt && \
-  echo "${FILEBEAT_SHA1}  filebeat.tar.gz" | sha1sum -c - && \
-  tar xzvf filebeat.tar.gz && \
-  cd filebeat-* && \
-  cp filebeat /bin && \
-  cd /opt && \
-  rm -rf filebeat* && \
-  apt-get purge -y wget && \
-  apt-get autoremove -y && \
-  apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    apt-get update && \
+    apt-get install -y wget && \
+    apt-get clean
 
+#install and configure filebeat
 RUN mkdir /filebeat /filebeat/config /filebeat/data && \
-    chmod a+w /filebeat/data
+    chmod a+w /filebeat/data && \
+    wget https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-${FILEBEAT_VERSION}-linux-x86_64.tar.gz -O /opt/filebeat.tar.gz && \
+    cd /opt && \
+    echo "${FILEBEAT_SHA1}  filebeat.tar.gz" | sha1sum -c - && \
+    tar xzvf filebeat.tar.gz && \
+    cd filebeat-* && \
+    cp filebeat /bin && \
+    cp filebeat.template.json /filebeat && \
+    mv module /filebeat
+
+#clean up
+RUN cd /opt && \
```

Comment:
You should do the cleanup in the same `RUN` as the fetching and installing stuff. Or you lose it's purpose because of how Docker caches the layers. Actually you should do just one single `RUN` to keep the image small.